scripts/jenkins/ardana/ansible/roles/ardana_deploy/tasks/main.yml

```diff
@@ -93,3 +93,14 @@
     - not (ansible_scratch_plays | default({})) is failed
   loop_control:
     label: "{{ item.play }}"
+
+- name: Get public vip
+  shell: grep  'public-KEY-API-extapi' /etc/hosts | awk {'print $1'}
```

Comment:
Meera, is this really the official way of configuring an external name ? Could you add a link to the official documentation where this is explained, please ?

Reply:
Stefan, Yes we would want to make this configurable. We would want to test cases where we have external-name configured and also not configured.
Regarding how I'm fetching the pubic vip, I can use the read_group_vars.yml playbook to fetch the public vip. The read_group_vars.yml was introduced to support fetching information needed to run some keystone tests. So we were planning to run that playbook only for keystone tests.